packages/riipen-ui-docs/src/pages/components/badge/badge.md

```diff
@@ -43,3 +43,9 @@ You can use the `anchor` property to move the badge to any corner of the wrapped
 You can use the `color` property choose the badge color.
 
 {{"demo": "pages/components/badge/Color.js"}}
+
+## Size
+
+You can use the `size` property to choose the badge size, if the `variant` is `standard`.
```

Comment:
Maybe double-check that this sounds nice.

Reply:
ah yes my language skills are well known.
![image](https://user-images.githubusercontent.com/1365762/96192067-20cf4700-0efa-11eb-92ca-1d658679a937.png)

Reply:
Sounds ok maybe something like?
```
With the `standard` `variant`, use the `size` property to set the badge size.
```
Or
```
Use the `size` property to choose the badge size. Only works with the `standard` `variant`
```
If you dont like those keeping as works as well